earthly/rust/Earthfile

```diff
@@ -49,6 +49,9 @@ rust-base:
     # Install a nightly toolchain which matches.
     RUN rustup toolchain install nightly --component miri --component rust-src --component rustfmt --component clippy
 
+    # Add a target for wasm.`
+    RUN rustup target add wasm32-unknown-unknown --toolchain nightly
```

Comment:
No no, this target configuration should be a part of the each rust project config.
You should try to put target and channel specification to the rust-toolchain.toml file.

Reply:
Okay, then the PR is dismissed.